features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,16 @@
-# put step definitions in this file
\ No newline at end of file
+Given(/^I visit the site$/) do
+  visit '/'
+end
+
+Then(/^I would like to see "([^"]*)"$/) do |text|
+  expect(page).to have_content text
+end
+
+When(/^I visit the name page for "([^"]*)"$/) do |name|
+  visit '/name/Amber'
```

Comment:
You save the variable but you are not using it. Why?

Reply:
I got this error, Your block takes 0 arguments, but the Regexp matched 1 argument. When I added it I got my code passing.  Looking into this again now.

Reply:
Am I getting a false positive? I am not sure what I am missing - arg = name because we are looking for the user's name on the page and the user = params[:name] is Amber.

Reply:
How about 
```
When(/^I visit the name page for "([^"]*)"$/) do |name|
   visit "/name/#{name}"
end
```

Reply:
Thank you, didn't know you could use string interpolation with the root paths :)